src/atoms/Identicon/Identicon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import styled from '_styled-components';
+import makeBlockie from 'ethereum-blockies-base64';
```

Comment:
Let's move `ethereum-blockies-base64` above `react` since it's an external package.

Reply:
Gotcha. I'll keep this in mind for future changes as well.